packages/transaction-controller/tsconfig.json

```diff
@@ -1,7 +1,8 @@
 {
   "extends": "../../tsconfig.packages.json",
   "compilerOptions": {
-    "baseUrl": "./"
+    "baseUrl": "./",
+    "target": "ES2022"
```

Comment:
This is necessary now because we're using BigInts in a test. This only affects local compilation of this package; the published version will still use a target of ES6.